server/routes/pages/index.js

```diff
@@ -6,6 +6,7 @@ module.exports = (app) => {
   app.get('/', handlePageRequest);
   app.get('/login', handlePageRequest);
   app.get('/about', handlePageRequest);
+  app.get('/tos', handlePageRequest);
```

Comment:
I'm re-writing how routes work, the conflicts will be minor though; I'm adding more robust support including contexts that include route actions.